src/main/resources/org/jenkinsci/plugins/workflow/steps/MailStep/help.html

```diff
@@ -1,3 +1,15 @@
 <div>
     Simple step for sending email.
 </div>
+<br>
+<p>Example:
+<pre><code>
+    mail bcc: '',
+         body: '',
+         cc: '',
+         from: '',
+         replyTo: '',
+         subject: '',
+         to: ''
```

Comment:
Adds nothing beyond what you would get from the snippet generator already, with much more context.